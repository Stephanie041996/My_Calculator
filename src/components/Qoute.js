import React from 'react';

const Qoute = () => (
  <p className="qoute">
    Mathematics is not about numbers, equations, computations, or algorithms:
    it is about understanding. â€“William Paul Thurston
  </p>

);
export default Qoute;
