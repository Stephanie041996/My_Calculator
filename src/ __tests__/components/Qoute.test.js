import React from 'react';
import renderer from 'react-test-renderer';
import Qoute from '../../components/Qoute';

describe('Qoute', () => {
  it('should render qoute', () => {
    const tree = renderer.create(<Qoute />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
