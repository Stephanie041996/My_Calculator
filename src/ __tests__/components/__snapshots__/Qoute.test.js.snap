// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Qoute should render qoute 1`] = `
<p
  className="qoute"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</p>
`;
